hackr/web.py

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-import requests
 import json
-from xmljson import badgerfish as bf
 from xml.etree.ElementTree import tostring
 
+import requests
+from xmljson import badgerfish as bf
```

Comment:
`xmljson` needs to be mentioned in the requirements file. Can you please update it?